.github/workflows/ios-canary.yml

```
@@ -0,0 +1,118 @@
+  cancel-in-progress: true
+
+jobs:
+  ios-canary:
+    name: "${{ github.event_name == 'push' && 'ios-canary-release' || 'ios-canary-ad-hoc' }}"
+    runs-on: [self-hosted, macOS, ios-canary]
+    timeout-minutes: 35
+    env:
+      SYSTEMSCULPT_RUNTIME_SMOKE_LICENSE_KEY: ${{ secrets.SYSTEMSCULPT_E2E_LICENSE_KEY }}
+      SYSTEMSCULPT_RUNTIME_SMOKE_SERVER_URL: ${{ secrets.SYSTEMSCULPT_RUNTIME_SMOKE_SERVER_URL }}
+      SYSTEMSCULPT_IOS_SYNC_CONFIG_B64: ${{ secrets.SYSTEMSCULPT_IOS_SYNC_CONFIG_B64 }}
+      IOS_CANARY_DIR: ${{ runner.temp }}/systemsculpt-ios-canary/${{ github.run_id }}-${{ github.run_attempt }}
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)